command/select.go

```diff
@@ -0,0 +1,163 @@
+package command
+
+import (
+	"context"
+	"fmt"
+	"os"
+
+	"github.com/urfave/cli/v2"
+
+	"github.com/peak/s5cmd/log/stat"
+	"github.com/peak/s5cmd/parallel"
+	"github.com/peak/s5cmd/storage"
+	"github.com/peak/s5cmd/storage/url"
+)
+
+var selectHelpTemplate = `Name:
+	{{.HelpName}} - {{.Usage}}
+
+Usage:
+	{{.HelpName}} [options] argument
+
+Options:
+	{{range .VisibleFlags}}{{.}}
+	{{end}}
+Examples:
+	01. Search for all JSON objects with the foo property set to 'bar' and spit them into stdout
+		 > s5cmd {{.HelpName}} --compression-type gzip --query "SELECT * FROM S3Object s WHERE s.foo='bar'" s3://bucket/*
+`
+
+var selectCommandFlags = []cli.Flag{
```

Comment:
It might make sense to explicitly require the format here - I'm not sure I have the momentum to add full support for parquet and CSV formats, but I could at least add the flag? Let me know if CSV + parquet support seem mandatory for landing this.

Reply:
I think we can do the JSON first and add others later, one by one. CSV is very common but I don't think its critical to support it at first pass.

A flag sounds good to me.